src/index.ts

```diff
@@ -51,10 +61,15 @@ export function load(app: Application) {
     });
 
     app.converter.addUnknownSymbolResolver((declaration) => {
+        const validSources = [
+            ...app.options.getValue("additionalModuleSources"),
+            "typescript",
```

Comment:
Shouldn't we have `@webgpu/types` here?

Reply:
The concern I had with listing it explicitly was in the case that another plugin is intended to handle certain modules. For example, MDNs documentation is actually not exhaustive for `@webgpu/types`, however the maintainers have their own typedoc generated for the spec.

MDN not having the full spec is likely because the spec is still experimental and new (pure speculation); it will eventually be added to the `dom` module. This leads me to believe that perhaps moduleSources that are not caught here are likely not quite stable -- you may know more about that than I.

If one wanted to use another plugin to link to external documentation (not on MDN) instead then would this cause a conflict? Do plugins execute in the order they are specified? If so then this point is moot, and it should indeed be added here.